.github/workflows/Deploy.yaml

```
@@ -90,3 +119,38 @@ jobs:
             Write-Host "::Error::Error deploying repositories. The error was $($_.Exception.Message)"
             exit 1
           }
+
+      - name: Calculate Release Notes
+        if: github.repository_owner == 'microsoft'
+        run: |
+          $errorActionPreference = "Stop"; $ProgressPreference = "SilentlyContinue"; Set-StrictMode -Version 2.0
+          $releaseNotesFile = Join-Path $env:GITHUB_WORKSPACE "RELEASENOTES.md"
+          $releaseNotes = (Get-Content -Encoding utf8 -Path $releaseNotesFile) -join "`n"
+          $lastVersion = $releaseNotes.indexof("`n## ")
+          $releaseNotes = $releaseNotes.Substring(0, $lastVersion)
+          Add-Content -encoding UTF8 -Path $env:GITHUB_ENV -Value "ReleaseNotes=$([Uri]::EscapeDataString($releaseNotes))"
+
+      - name: Create release
+        if: github.repository_owner == 'microsoft' && github.event.inputs.createRelease
```

```suggestion
        if: github.repository_owner == 'microsoft' && github.event.inputs.createRelease == 'true'
```

Not sure if I remember this correctly, but aren't the boolean inputs to GitHub either 'true' or 'false' (as strings)? 

For example in E2E do: 
`github.event.inputs.runUpgradeTests == 'true'`